scalableminds/webknossos
# Advanced search for comments

I replaced the search in comments tab with the new advanced search. I also added the `enter+shift` shortcut to cycle backward through the results. Previously only the `enter` shortcut to cycle forward existed. As far as I can remember the `enter+shift` shortcut is common for all kind of search functionalities and therefore should be more or less discoverable.

### URL of deployed dev instance (used for testing):
- https://advancedsearchforcomments.webknossos.xyz

### Steps to test:
- Open a skeleton/hybrid tracing and create some trees with nodes.
- Then open the comment tab and add comments to the nodes.
- Then open up the search via the shortcut (`shift+crtl+f`) or/and via the button. Both should work out fine.
- Then just use the search as any other search. Everything should work as expected. This also counts for the `enter`(select next) and `shift+enter` (select previous) shortcut.


### Issues:
- fixes #4229 

------
